messagebird/base_list.py

```diff
@@ -43,3 +43,7 @@ def items(self, value):
             items.append(self.itemType().load(item))
 
         self._items = items
+
+    def __str__(self):
+        items_count = 0 if self.items is None else len(self.items)
+        return str(self.__class__) + " with %d items.\n" % items_count
```

Comment:
Maybe `return  "%s with %d items.\n" % (str(self.__class__), items_count)` would be a bit nicer